pkg/aks/components/AksNodePool.vue

```diff
@@ -5,7 +5,7 @@ import { mapGetters } from 'vuex';
 import { _CREATE, _VIEW } from '@shell/config/query-params';
 import type { AKSDiskType, AKSNodePool, AKSPoolMode } from '../types/index';
 import LabeledInput from '@components/Form/LabeledInput/LabeledInput.vue';
-import LabeledSelect from '@shell/components/form/LabeledSelect.vue';
+import LabeledSelect from '@shell/components/form/LabeledSelect/index.vue';
```

Comment:
does this need the `index.vue`? on other places it wasn't needed

Reply:
TS components need the explicit `index.vue`, JS is fine with just the folder name